FrostyEditor/Utils/Config.cs

```diff
@@ -211,14 +211,11 @@ public static IEnumerable<string> GameProfiles
     /// </summary>
     /// <param name="profile">The profile name.</param>
     /// <param name="gamePath">The path to the game.</param>
-    public static void AddGame(string profile, string gamePath)
+    public static bool AddGame(string profile, string gamePath)
```

Comment:
can u add the return thing to the comments above, just have it say sth like `True if the game was added successfully`

Reply:
i meant like /// <returns>True if the game was added successfully.</returns>

Reply:
sorry for that